app/moves/middleware.js

```diff
@@ -44,6 +62,24 @@ module.exports = {
     res.locals.moveDate = format(parsedDate, moveDateFormat)
```

Comment:
Is this now even really used anywhere/needed anymore? We're basically replacing the need for this with the range I think.

The only place I can see this being used is in the existing header. But you could replace this with `setDateRange` and use `dateRange | first` again instead of `moveDate`.